### ISSUE TITLE
Add support for vix/vixen pronouns

### ISSUE THREAD

**Issue by Foxlass** [cyyynthia/pronoundb.org#96]
I don't use any of the currently listed pronouns that are on PronounDB (I use vix/vixen), so it would be nice to have support for them.

**Comment by 9021007** [comment]
Please see the README:
![image](https://github.com/cyyynthia/pronoundb.org/assets/24487638/725c25dc-d4e2-42e4-9943-b797b241ecb0)

**Comment by cyyynthia** [comment]
As mentioned above, neopronouns support is addressed in the README. There is underlying work going on to make adding more sets viable (rework the way pronouns are represented internally), but for now with current infrastructure it's not possible to add new sets easily.

As with all the other similar issues, I'll close this as I don't want the tracker to be full of duplicates that are already addressed in the README file. Once the internal restructuring is complete, I will look into adding more sets.
